packages/web-components/src/global/_settings.scss

```diff
@@ -0,0 +1,3 @@
+@use "uswds-core" with (
+  $theme-image-path: '../../../../../node_modules/@uswds/uswds/dist/img/',
+);
```

Comment:
Great job figuring this out. I'm going to add a note here for future us to explore this override method for fonts as well related to this issue:

https://github.com/department-of-veterans-affairs/vets-design-system-documentation/issues/1411

USWDS v3 Reference: https://github.com/uswds/uswds/blob/b715c487ceb7fa18075189ee79861c482c5b61a6/README.md#set-the-base-asset-paths-fonts-and-images